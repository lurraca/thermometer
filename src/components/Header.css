.title {
    text-align: center;
    vertical-align: middle;
}

h1.app-name {
    line-height: 10px;
}

h2.app-tagline {
    color: #6cf;;
}
