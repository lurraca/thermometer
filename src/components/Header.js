import React, { Component } from 'react';

import './Header.css';

class Header extends Component {
  render() {
    return(
      <header>
        <div className="title">
          <h1 className="app-name">Thermometer</h1>
          <h2 className="app-tagline">Your premium source for weather details</h2>
        </div>
      </header>
    )
  };
};

export default Header;