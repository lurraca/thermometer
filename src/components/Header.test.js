import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header'

it('Header renders correctly', () => {
  const renderedComponent = shallow(
    <Header />
  );

  expect(renderedComponent).toMatchSnapshot();
})