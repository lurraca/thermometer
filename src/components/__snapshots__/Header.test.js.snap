// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <header>
    <div
        className="title"
    >
        <h1
            className="app-name"
        >
            Thermometer
        </h1>
        <h2
            className="app-tagline"
        >
            Your premium source for weather details
        </h2>
    </div>
</header>,
  "nodes": Array [
    <header>
      <div
            className="title"
      >
            <h1
                  className="app-name"
            >
                  Thermometer
            </h1>
            <h2
                  className="app-tagline"
            >
                  Your premium source for weather details
            </h2>
      </div>
</header>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Header />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Header {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <header>
          <div
                    className="title"
          >
                    <h1
                              className="app-name"
                    >
                              Thermometer
                    </h1>
                    <h2
                              className="app-tagline"
                    >
                              Your premium source for weather details
                    </h2>
          </div>
</header>,
        "_debugID": 2,
        "_renderedOutput": <header>
          <div
                    className="title"
          >
                    <h1
                              className="app-name"
                    >
                              Thermometer
                    </h1>
                    <h2
                              className="app-tagline"
                    >
                              Your premium source for weather details
                    </h2>
          </div>
</header>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Header />,
}
`;
